assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_list_assignments.vm

```diff
@@ -239,7 +245,8 @@
 								#end
 							</td>						
 							<td headers="title" scope="row">
-									#if (($!allowAddAssignment || $!allowUpdateAssignment || $!service.allowGradeSubmission($assignment.getReference())) && $!view.equals('lisofass1'))
+									## These functions could be improved to improve the instructor synoptic view
+									#if (!$!isSynopticTool && ($!allowAddAssignment || $!allowUpdateAssignment || $!service.allowGradeSubmission($assignment.getReference())) && $!view.equals('lisofass1'))
```

Comment:
Is this change needed as you're setting `$allowAddAssignment` to `false` at the top of the template.